.github/workflows/e2e-all-k8s.yml

```diff
@@ -0,0 +1,40 @@
+---
+name: End to End All K8s
+
+on:
+  pull_request:
+    types: [labeled, opened, synchronize, reopened]
+  schedule:
+    - cron: "0 0 * * 6"
+
+jobs:
+  e2e:
+    name: E2E
+    if: contains(github.event.pull_request.labels.*.name, 'e2e-all-k8s')
+    timeout-minutes: 30
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        globalnet: ['', 'globalnet']
+        k8s_version: ['1.17', '1.20', '1.21', '1.22', '1.23']
```

Comment:
Should we drop 1.17?

Reply:
No, 1.17 is currently the "last thought to be working version", which we want to keep testing to maintain https://github.com/submariner-io/submariner-operator/blob/devel/pkg/version/version.go#L34. See the proposed docs about all of this: https://github.com/submariner-io/submariner-website/pull/643

Reply:
This isn't aligned with the docs though

> Submariner's policy is to support all versions upstream-Kubernetes supports and no EOL versions.

Also didn't we agree not to to officially test these old versions, so that we don't give a false semblance of supporting them?

Reply:
> This isn't aligned with the docs though
> 
> > Submariner's policy is to support all versions upstream-Kubernetes supports and no EOL versions.
> 
> Also didn't we agree not to to officially test these old versions, so that we don't give a false semblance of supporting them?

I thought so too There was an issue as well (can't find it right now though) to track this ans was discussed in one of the meetings.

Reply:
IIRC from the discussions, we wanted to distinguish the minimum version we *support*, and the version below which we prevent installation. There’s a difference between “we’ll let you install this, but we won’t provide support” (for some value of “support” anyway in an open source project), and “we know this will break, so we’ll prevent you from doing it” (unless you provide the appropriate flag).

Reply:
@mkolesnik I'd like to go ahead and merge this, so we don't have to rebase and run all the jobs again. Does Stephen's answer clarify things?